docs/source/support/must-gather.md

```diff
@@ -0,0 +1,63 @@
+# Gathering data with must-gather
+
+`must-gather` is an embedded tool in Scylla Operator that helps collecting all the necessary info when something goes wrong.
+
+The tool talks to the Kubernetes API, retrieves a predefined set of resources and saves them into a folder in your current directory.
+By default, all collected Secrets are censored to avoid sending sensitive data inside the archive.
+That said, you can always review the archive before you attach it to an issue or your support request.
+
+Given it needs to talk to the Kubernetes API, at very least, you need to supply the `--kubeconfig` flag with a path to the kubeconfig file for your Kubernetes cluster, or set the `KUBECONFIG` environment variable.
+
+## Running must-gather
+
+There is more than one way to run `must-gather`.
+Here are some examples of how you can run the tool.
+
+All examples assume you have exported `KUBECONFIG` environment variable that points to a kubeconfig file on your machine.
+If not, you can run this command to export the common default location.
+Please make sure such file exists.
+
+```bash
+export KUBECONFIG=~/.kube/config
+ls -l "${KUBECONFIG}"
+```
+
+```note::
+   There can be slight deviations in the arguments for your container tool, depending on the container runtime, whether you use SELinux or similar factors.
+```
+
+### Podman
+```bash
+podman run -it --pull=always --rm -v="${KUBECONFIG}:/kubeconfig:Z,ro" -v="$( pwd ):/workspace:Z" --workdir=/workspace docker.io/scylladb/scylla-operator:latest must-gather --kubeconfig=/kubeconfig
```

Comment:
Will this actually work if your kubeconfig points to an EKS or GKE cluster (which is probably going to be a case with most of our users)? Iirc when I tried it with EKS it complained about missing authentication plugins for client-go.

Reply:
hm, I am using token based kubeconfigs for both of them so while I've tried, it was a different path. worth looking into, thx.

Reply:
added the extra steps